net/yazproxy/files/dynamic_lookup-11.patch

```diff
@@ -1,26 +1,31 @@
 Recognize macOS 11 and later.
 https://debbugs.gnu.org/cgi/bugreport.cgi?bug=44605
 https://github.com/indexdata/yazproxy/issues/21
---- configure.orig	2024-05-23 14:09:36.000000000 -0500
-+++ configure	2024-05-23 14:10:02.000000000 -0500
-@@ -7923,16 +7923,11 @@
-       _lt_dar_allow_undefined='${wl}-undefined ${wl}suppress' ;;
+--- configure	2022-08-18 16:32:05
++++ configure	2024-06-06 12:59:04
+@@ -9083,18 +9083,13 @@
+       _lt_dar_allow_undefined='$wl-undefined ${wl}suppress' ;;
      darwin1.*)
-       _lt_dar_allow_undefined='${wl}-flat_namespace ${wl}-undefined ${wl}suppress' ;;
+       _lt_dar_allow_undefined='$wl-flat_namespace $wl-undefined ${wl}suppress' ;;
```

Comment:
There's no need to change this line. The curly braces around `{wl}` are not necessary in these two instances but are not harmful either. There have been many changes in libtool over the years and thus many subtle variations in the dynamic_lookup-11.patch files in MacPorts. Our goal is not to completely change a project's code to match the latest libtool, but adapt the patch so that it makes the minimal changes to accomplish the goal.